core/src/main/java/org/apache/accumulo/core/clientImpl/ThriftScanner.java

```diff
@@ -275,12 +275,12 @@ static <T> Optional<T> waitUntil(Supplier<Optional<T>> condition, Duration maxWa
         .incrementBy(100, MILLISECONDS).maxWait(1, SECONDS).backOffFactor(1.5)
         .logInterval(3, TimeUnit.MINUTES).createRetry();
 
-    long startTime = System.nanoTime();
+    Timer startTime = Timer.startNew();
     Optional<T> optional = condition.get();
     while (optional.isEmpty()) {
       log.trace("For tableId {} scan server selector is waiting for '{}'", tableId, description);
 
-      var elapsedTime = Duration.ofNanos(System.nanoTime() - startTime);
+      var elapsedTime = startTime.elapsed();
 
       if (elapsedTime.compareTo(timeoutLeft) > 0) {
```

Comment:
```suggestion
      if (startTime.hasElapsed(timeoutLeft)) {
```

There are a few places where `Timer.hasElapsed()` can be used instead of needing to grab the elapsed time via  `elapsed()` and then `compareTo()`.

I also think it makes the code read better if the vars are renamed to timer. In this case for example `startTime.hasElapsed()` gives off the wrong idea while reading it (in my head at least) vs. something like `waitTimer.hasElapsed()`.